gtsfm/loader/loader_base.py

```diff
@@ -290,6 +290,11 @@ def get_image_shapes(self) -> List[Tuple[int,int]]:
         N = len(self)
         return [self.get_image_shape(i) for i in range(N)]
 
+    def get_image_fnames(self) -> List[str]:
```

Comment:
I wonder why this is queried all at once, where as other properties like the shape and the image itself are queried for a certain index.

Reply:
We have functions like `get_image_shapes` too.